pom.xml

```diff
@@ -33,6 +33,7 @@
         <httpclient.version>4.5.13</httpclient.version>
         <ipaddress.version>5.3.1</ipaddress.version>
         <jackson.version>2.10.0</jackson.version>
+        <jackson.blackbird.version>2.12.5</jackson.blackbird.version>
```

Comment:
Can we up with `jackson.version` instead of just `blackbird` ?

Reply:
1. Why exactly did you increased version of blackbird? 
2. What is acceptance criteria of successful version increment for Jackson itself? From your perspective?

PS: Testing went like a charm with 2.12.5 version of jackson, also looked through change of Jackson, it looks they have no breaking changes.
PSS: https://docs.spring.io/spring-boot/docs/current/reference/html/index.html as we go this path, Spring boot 2.5.5 might be worth trying as dependencies state on the link. We might want to discuss options here.